client/app/components/Input/InputSelect.jsx

```diff
@@ -9,6 +9,7 @@ import type { Option } from './utils';
 export type Props = {
   id: string,
   name?: string,
+  aria?: any,
```

Comment:
Let's also add ariaLabel to `inputSelectProps` in `client/app/mocks/InputMocks.jsx` so that we're testing for it as well.